test/Feature/InsertSpec.hs

```diff
@@ -418,8 +412,8 @@ spec actualPgVersion = do
 
     context "with unicode values" $
       it "succeeds and returns usable location header" $ do
-        let payload = [json| { "a":"圍棋", "b":"￥" } |]
-        p <- request methodPost "/no_pk"
+        let payload = [json| { "k":"圍棋", "extra":"￥" } |]
+        p <- request methodPost "/simple_pk"
```

Comment:
I also get the same error, it's weird. But since the test was meant for `no_pk` Location, and we're changing this behavior, I think we can just delete this one. There's another unicode test below that is working.

Reply:
The other test is for `PATCH`, I think, and doesn't test the location header with unicode in it. This one still does, because the changed table now has the unicode column as PK. So the test does have some value.

Since the expected output was the wrong way around (in regards to key order) - I now just changed the actual ouput's key order by adding a `select=extra,k`. Should pass now.